jeb_bearer.cfdg

```diff
@@ -0,0 +1,34 @@
+startshape LINES
+
+rule LINES {
+	LINE{r 45 hue 90 sat 1 brightness 0}
```

Comment:
I personally used "b" as an abbreviation for "brightness," and "s" as an abbreviation for "size".